UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -827,6 +827,62 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
             return string.Format(provider, format, formatArgs);
         }
 
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using <see cref="GlobalConfiguration.DefaultCulture" />.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format)
+        {
+            return ToString(format, GlobalConfiguration.DefaultCulture);
+        }
+
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using the specified format provider, or <see cref="GlobalConfiguration.DefaultCulture" /> if null.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <param name="formatProvider">Format to use for localization and number formatting. Defaults to <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format, IFormatProvider formatProvider)
+        {
+            formatProvider = formatProvider ?? GlobalConfiguration.DefaultCulture;
+
+            int digits = 0;
+            string formatString = format;
+
+            if(string.IsNullOrEmpty(formatString))
+                formatString = "G";
+
+            formatString = formatString.ToUpperInvariant();
+
+            if(formatString.StartsWith("A") || formatString.StartsWith("S"))
+            {
+                if(formatString.Length > 1 && !int.TryParse(formatString.Substring(1), out digits))
+                    throw new FormatException($"The {format} format string is not supported.");
+
+                formatString = formatString.Substring(0, 1);
+            }
+
+            switch(formatString)
+            {
+                case "G":
+                    return ToString(formatProvider, 2);
+                case "A":
+                    var abbreviations = UnitAbbreviationsCache.Default.GetUnitAbbreviations<AccelerationUnit>(Unit, formatProvider);
+                    return abbreviations[digits];
+                case "V":
+                    return Value.ToString(formatProvider);
```

Comment:
Maybe support significant digits after radix point here? `Mass.FromKilograms(15.3).ToString("V4")` could output `15.3000`.

Reply:
In the same spirit `"V####"` could maybe output "15.3", since `#` is used to omit trailing zeros after radix point (and leading zeros before radix point).

Reply:
That's what S does (must be given a number). You want both to do the same thing?

Reply:
I always wonder at what point do we stop with the strings and let the user just do a string.format("0.####", quantity.Value) ❓

Reply:
Fair points, I've had the same thought.
Doesn't `S` also include the abbreviation? Maybe I read it wrong.

Reply:
`Length.FromCentimeters(15.34000).ToString(null, 5)` => `15.34 cm`, so yes, it does the `#` thing.

So my proposal would be:
`Length.FromCentimeters(15.34000).ToString("V####")` to get `15.34` without the unit abbreviation.

It's valid on its own, but I agree with you, at some point people should just do standard/custom numeric formatting of the `Value` property. I can agree to don't do the `V####` usecase on that ground.